### ISSUE TITLE
Unhandled exception: c0000005

### ISSUE THREAD

**Issue by fhuewhd** [TheLegendOfMataNui/game-issues#294]
I'm using the Rebuilt version, version 1.0.1 if "versionop.txt" is accurate.

I'm using Linux, with PlayOnLinux.  (It's one of the games with an approved pre-made installer on PlayOnLinux's list https://www.playonlinux.com/en/app-3799-Lego_Bionicle_The_Legend_of_Mata_Nui.html , so it ought to work.)

It was working before - at least, I could get it to run, though I'm not sure there weren't some bugs.  
But now when I try to run it, I get a white screen (straight away, doesn't even start to display the intro) and two errors, "Unhandled exception: c0000005 at address 005dd477" and "Unhandled exception: c0000005 at address 7e7939d0".  (I'm not sure about the "5" at the end of "c0000005".  I copied that from somebody else's posting, but on mine the number runs off the end of the box.)

Mean anything to any of you?

**Comment by ShadowOfNight** [comment]
So to be clear, it worked previously and now does not?  When is the crash occurring, on startup or when you get to a menu, etc?  Have you tried updating the game?  Have you made any changes to your Linux install recently?

**Comment by fhuewhd** [comment]
Like I said, straight away.  I launch the program, I get a white screen.  
I don't know how you would "update the game".  
I installed it one night, it ran correctly, several times in fact, the next night, it didn't run.  I hadn't done anything to Linux in between that I remember.
I've discovered, though, that it does sometimes run.  In fact, I've been clicking rapidly on the program icon several times to launch several copies of the program, and sometimes one or more of them will have the menu instead of the white screen, and then I close the duplicate ones.  After that, it seems to run correctly, as far as I can tell so far!
Thanks for the reply.

**Comment by fhuewhd** [comment]
So far, even if the game is running normally before that, selecting Level 2 _always_ crashes it (white screen as before immediately on clicking "Level 2", doesn't go away if I wait a minute).  Level 3 has loaded successfully the one time I've tried it so far, though, so perhaps that's specific to Level 2 - I see there are several other reports about bugs in Level 2.

**Comment by ShadowOfNight** [comment]
@fhuewhd which version of Wine are you using, and which Linux distro do you have?  Level 2 has a known issue with Wine, which is why that's crashing.  We've been looking into it, so hopefully we'll have a fix out for the next patch.  In the meantime, Wine 7.0 at least seems to boot the game fine with no issues, whereas the modified Wine 6.21 included with Lutris seems to crash in similar fashion to your first post in this ticket.

**Comment by fhuewhd** [comment]
Zorin OS 12 (a variation of Ubuntu Xenial).  I _thought_ I was using Wine 6, but on closer inspection it seems to be Wine 5.0.3.  Am trying to sort that out.
That sounds like it might be to do with it, anyway.  Thanks for the reply!

**Comment by ShadowOfNight** [comment]
You're quite welcome!  Do keep us updated on how the new version of Wine behaves for you as well.  The interaction with level2 that crashes is unfortunately a bug with Wine that doesn't seem to be getting fixed, so we'll be making adjustments on our end to compensate and hopefully fix this in the next patch.

**Comment by fhuewhd** [comment]
I got Wine 7.0-rc5 installed via PlayOnLinux, and I've opened and closed the game several times since then and it's started first time each time!  (Except for an incident where it opened but not in full screen, while still being full size so that it was cut off.  But most often that hasn't been happening.)  I haven't tried Level 2 yet.

I haven't managed to find the discus yet, or work out how to get the last token from the rock-face, but that's a me problem, not a program problem!  :-D

There _is_ still some kind of error with the inventory, though, if somebody could take a look at that.  https://github.com/TheLegendOfMataNui/game-issues/issues/49
(It's a closed issue so I don't know whether anybody would see it unless I mentioned it elsewhere).
